values/aws-alb-ingress-controller/aws-alb-ingress-controller-expert.gotmpl

```diff
@@ -0,0 +1 @@
+{{- .Environment.Values | get "charts.aws-alb-ingress-controller._expert" dict | toYaml }}
```

Comment:
please remove all of these files and create a generic line of code and stay DRY

Reply:
Already tried that one but templating language is a bit limiting me here:
how to retrieve a key from an object by .Release.Name ?
The below line does not work:
{{- .Environment.Values | get "charts.{{ Release.Name  }}._expert" dict | toYaml }}
If you have any ideas the please write it down.

Reply:
```
{{- .Environment.Values | get (printf "charts.%s._rawValues" .Release.Name) dict | toYaml }}
```
Would this not work?

Reply:
brilliant !

Reply:
done

Reply:
> I haven't done a lot with this templating language yet so this might be totally useless, but I am reading the [documentation](https://masterminds.github.io/sprig/dicts.html) and it says you need to pass the dict function a key and value? But that would, of course, not make sense if it works for the functions without {{ Release.Name }}. Then maybe it gets passed an empty value somewhere?

@svatwork that dict without values is to give a default empty dict in case the first param of `get` can't find anything ;)